java/core/src/main/java/sleeper/core/statestore/FileInfoStore.java

```diff
@@ -132,6 +140,24 @@ void atomicallyUpdateJobStatusOfFiles(String jobId, List<FileInfo> fileInfos)
      */
     Map<String, List<String>> getPartitionToActiveFilesMap() throws StateStoreException;
 
+    /**
+     * Returns a list of files that are ready for garbage collection, i.e. there are no active file records referencing
+     * them and the last update time is before maxUpdateTime.
+     *
+     * @param maxUpdateTime The latest time at which a file can have been updated in order to be garbage collected
+     * @return an iterator of filenames with the matching status
```

Comment:
This method returns a Stream, not an iterator.

Reply:
I've corrected the comment.